### PR TITLE
Searching1

### PR DESCRIPTION
**Pull request by YehyaTamimi**

### CODE REVIEW

**Comment by DuaaH** [comment]
image

please add something here to tell the user there is no result

File: Js/search.js [hunk 1]
```diff
@@ -23,20 +28,49 @@ document.addEventListener('DOMContentLoaded', () => {
         viewFilter("rooms")
     });
 
-    document.querySelector(".home").addEventListener("click", () => {
-        window.location.href = "/html/index.html";
-    })
+
+    document.querySelector(".home").addEventListener("click", gotoHomePage);
+
+    let input = document.querySelector(".search-input");
```
Comment (DuaaH) [inline Js/search.js:34]: use const instead of let

**Comment by DuaaH** [comment]
please keep the searched value that I entered on the first page on the second page as well 
and I don't see the no result msg on the second screen if I write the wrong value
image
image

**Comment by YehyaTamimi** [comment]
> please keep the searched value that I entered on the first page on the second page as well and I don't see the no result msg on the second screen if I write the wrong value image image


fixed

**Comment by DuaaH** [comment]
note: when adding the api instead of json file, you need to add a loader until we got the response

**Pull request merged**